ADALiOS/ADALiOS/ADAuthenticationRequest+WebRequest.m

```diff
@@ -48,8 +50,17 @@ - (void)executeRequest:(NSString *)authorizationServer
      {
          //Prefill the known elements in the item. These can be overridden by the response:
          ADTokenCacheStoreItem* item = [ADTokenCacheStoreItem new];
-         item.resource = _resource;
+         ADAuthenticationSettings* data = [ADAuthenticationSettings sharedInstance];
          item.clientId = _clientId;
+
+         if (data.policy){
+             
+             item.policy = data.policy;
+             
+             
+         }
```

Comment:
I didn't see that we had _policy floating around. I was learning the code as I was prototyping. I removed this and instead rely on _policy.